src/pages/signIn/style.jsx

```diff
@@ -0,0 +1,20 @@
+import styled from "@emotion/styled"
+
+export const SignInLayout = styled.div`
+  height: 100%;
+  width: 100%;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+`
+
+export const BtnWrapper = styled.div`
+  width: 100%;
+  margin: 10% 0;
+
+  display: flex;
+  justify-content: spacee-between;
```

Comment:
```suggestion
  justify-content: space-between;
```

오탈자가 있는데 원하는 대로 구현 되셨나요..?

Reply:
어쩐지 왜 의도대로 안되나 싶었습니다.. 찾아주셔서 김사합니다!